Test failure looks real
```
FAIL: corehq.blobs.tests.test_init:test_get_blobdb.test_get_blobdb{'msg': u"shared drive path is not a directory: '.*/file'$", 'root': u'file'}
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 337, in test
    return test_func(self, **args)
  File "/mnt/commcare-hq-ro/corehq/blobs/tests/test_init.py", line 38, in test_get_blobdb
    mod.get_blob_db()
  File "/usr/local/lib/python2.7/contextlib.py", line 35, in __exit__
    self.gen.throw(type, value, traceback)
  File "/vendor/lib/python2.7/site-packages/testil.py", line 249, in raises
    '%r does not match %r' % (msg.pattern, str(err))
AssertionError: u"shared drive path is not a directory: '.*/file'$" does not match "cannot initialize blob db: shared drive path is not a directory: u'/tmp/tmp9MPvb_/file'"
```

--- a/corehq/blobs/tests/test_init.py
+++ b/corehq/blobs/tests/test_init.py
@@ -15,14 +15,14 @@ from settingshelper import SharedDriveConfiguration
 @generate_cases([
     dict(root=None, msg=r"invalid shared drive path: None$"),
     dict(root=b"", msg=r"invalid shared drive path: ''$"),
-    dict(root="file", msg=r"shared drive path is not a directory: '.*/file'$"),
+    dict(root=b"file", msg=r"shared drive path is not a directory: '.*/file'$"),
     dict(blob_dir=None, msg="blob_dir is empty or not configured"),
     dict(blob_dir="", msg="blob_dir is empty or not configured"),
 ])
 def test_get_blobdb(self, msg, root=True, blob_dir=None):
     with tempdir() as tmp:
-        if root == "file":
-            tmp = join(tmp, "file")
+        if root == b"file":
+            tmp = join(tmp, b"file")
             with open(tmp, "w") as fh:
                 fh.write("x")
         conf = SharedDriveConfiguration(

